src/app/containers/Copyright/index.jsx

```diff
@@ -12,7 +12,7 @@ const CopyrightContainer = ({ children }) => {
       {imageCopyrightOffscreenText ? (
         <VisuallyHiddenText>{imageCopyrightOffscreenText}</VisuallyHiddenText>
       ) : null}
-      {children}
+      <span lang="en-gb">{children}</span>
```

Comment:
```suggestion
      <span lang="en-GB">{children}</span>
```
We should only add the lang attribute of `en-GB` here if the article's language value is not `en-GB`. 

We can add the language to the Service configuration files and use the value from there.
e.g. 
the News config can have `lang: 'en-GB',` added https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/news.js#L6
likewise the Persian config can have `lang: 'fa',` added
https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/persian.js#L6

This lang attribute for the article can be fetched via useContext avoce, just like the `imageCopyrightOffscreenText` is currently pulled in. 

Then we'll only add `lang="en-GB"` if this language value is not already `en-GB`.

Reply:
@sareh should I then create langContext to fit this case?